app/models/manageiq/providers/container_manager.rb

```diff
@@ -1,7 +1,5 @@
 module ManageIQ::Providers
   class ContainerManager < BaseManager
-    include HasMonitoringManagerMixin
-    include HasInfraManagerMixin
```

Comment:
this defined `ensure_infra_manager`

may want to remove that line lower on

Reply:
I don't see `ensure_infra_manager` referenced anywhere outside of `HasInfraManagerMixin`, did I just miss it?